dev.nuculabs.ImageTagger.yaml

```diff
@@ -0,0 +1,42 @@
+id: dev.nuculabs.ImageTagger
+runtime: org.freedesktop.Platform
+version: v1.3.0
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/image/bin/ImageTagger
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to save files locally
+  - --filesystem=xdg-pictures
+modules:
+  - name: Image Tagger
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/image/
+      - cp -R image/bin /app/bin/image/
+      - cp -R image/lib /app/bin/image/
+      - cp -R image/conf /app/bin/image/
+      - cp -R image/legal /app/bin/image/
+      - cp -R image/release /app/bin/image/
+      - chmod +x -R /app/bin/image/lib/ # make libs executable so xdg-open will work
+      # Desktop Integration
+      - mkdir -p bin /app/share/{applications,icons/hicolor/64x64/apps,metainfo}
+      - mv ${FLATPAK_ID}.desktop      /app/share/applications/${FLATPAK_ID}.desktop
+      - mv ${FLATPAK_ID}.png          /app/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
+    sources:
+      - type: archive
+        url: https://github.com/dnutiu/ImageTagger/releases/download/v1.3/ImageTagger-linux-1.3.zip
+        sha256: 0f086e6a738b3d59e3d05cce9174316d95886e50278c03e5b452a67fd264ea40
+        only-arches: [ x86_64 ]
+      - type: file
+        path: dev.nuculabs.ImageTagger.png
+      - type: file
+        path: dev.nuculabs.ImageTagger.desktop
+    dependencies:
```

Comment:
which part of the documentation did tell you this was the way to go?

Reply:
I've used Flat Seal to inspect permissions and I wanted to get a version tag, since I couldn't find it in the documentation I've tried several things and I've forgot this here. Thanks for pointing it out!

Reply:
I don't see the relation between what you are saying and "dependencies" Also the documentation is quite comprehensive, and there is nop mention of `dependencies` as a manifest property anywhere.